src/atom1.ts

```diff
@@ -11,10 +11,17 @@ import { sanitize } from "./utils";
 export default (ins: Feed) => {
   const { options } = ins;
 
+  const feedAttrs: Record<string, string> = { xmlns: "http://www.w3.org/2005/Atom" };
+  if (ins.options.language) {
+    // Atom uses the reserved "xml:" namespace for language;
+    // no extra xmlns declaration is required.
+    feedAttrs["xml:lang"] = sanitize(ins.options.language) || ins.options.language;
```

Comment:
Why fall back to the unsanitized value if the sanitization fails? Wouldn't it be better to throw an error, or log a warning and omit the `language` field instead?